LondonBritishLibrary/orient/BLorient604.xml

```diff
@@ -33,43 +34,234 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <altIdentifier>
                             <idno>Wright 291</idno>
                         </altIdentifier>
-                        
+
                     </msIdentifier>
-                    
+
                     <msContents>
-                        <summary/>
-                        
-                        
-                        
-                        
+
+    <msItem xml:id="ms_i1">
+    <locus from="5r"/>
+    <title type="complete" xml:lang="en" ref="LIT1293Dersan">Discourse of <persName ref="PRS9486Theophil">Theophilus of Alexandria</persName></title>
+    <incipit xml:lang="gez">ይቤ፡ በእንተ፡ እግዝእትነ፡ ማርያም፡ ድንግል፡ ወበእንተ፡ ቤት፡ ዘኀደረት፡ ውስቴቱ፡ ምስለ፡ ፍቁር፡ ወልዳ፡ ኢየሱስ፡ ክርስቶስ፡ በደብረ፡ ቍስቋም፡
+    ዘመልዕልተ፡ ደብረ፡ አመ፡ መጽኡ፡ ብሔረ፡ ግብጽ፡ እምፍርሃተ፡ ሄሮድስ፡ ንጉሥ። ወበእንተዝ፡ ወፅአ፡ ሊቀ፡ ጳጳሳት፡ ወበጽሐ፡ ወርእየ፡ ዘንተ፡ ኵሎ፡ ተአምረ። እስመ፡ ንጉሥ፡ በስጣዮስ፡ ክርስቲያናዊ፡
+    ወሀቦ፡ መራኁተ፡ ኵሉ፡ ቤተ፡ አማልክት፡ ዘብሔረ፡ ግብጽ፡ ከመ፡ ይንስቶሙ፤ ወይንሣእ፡ ኵሎ፡ ንዋዮሙ። ወይሕንፅ፡ አብያተ፡ ክርስቲያናት፡ ውስተ፡ ኵሉ፡ አድያሚሃ፡ ለግብፅ። ወሀለዉ፡ ኤጲስ፡ ቆጶሳት፡
+  ምስሌሁ። ወዜነውዎ፡ ክብረ፡ ለውእቱ፡ ደብር፡ ወጸጋ፡ ዘቤተ፡ ክርስቲያን፡ ዘላዕሌሁ። ወፈቀደ፡ ይሑር፡ ኀቤሁ፡ ከመ፡ ይንሣእ፡ በረከቶ፡ ወትንብልናሁ፡ ወበጽሐ፡ እንዘ፡ ዓዲ፡ ሠሉስ፡ መዋዕል፡ ለተዝካረ፡ በዓሉ፡
+  ለውእቱ፡ ደብረ። ወአስተብቍዕዎ፡ ኤጲስ፡ ቆጶሳት፡ ወመነኮሳት፡ ይንበር፡ ምስሌሆሙ፡ እስከ፡ የኃልፍ፡ በዓለ፡ ለቅድስት፡ ማርያም፡ ወኍልቆሙ፡ ፫፻ወነበረ፡ ምስሌሆሙ፡ በሰላመ፡ እግዚእነ፡ አሜን።</incipit>
+  <note>To be read on the 6th of Ḫǝdār</note>
+  <note>ዘይትነበብ፡ አመ፡ ፯ለወርኃ፡ ኅዳር፡ ዘከመ፡ ኀደረት፡ እግዝእትነ፡ ውስተ፡ ደብረ፡ ቍስቋም።</note>
```

Comment:
the element note is not appropriate here

Reply:
What is your suggestion?

Reply:
Sth like: `<note>The instruction on the reading of the text on the 6th of Ḫǝdār and the title are written in the upper margin <foreign xml:lang='gez'><hi rend="rubric"><add place="top">ዘይትነበብ፡ አመ፡ ፯ለወርኃ፡ ኅዳር፡ ዘከመ፡ ኀደረት፡ እግዝእትነ፡ ውስተ፡ ደብረ፡ ቍስቋም።</add></hi><foreign/></note>`